README.md

```diff
@@ -66,4 +91,53 @@ WifiManager.getCurrentWifiSSID()
 	console.log('Cannot get current SSID!')
 })
 ```
-  
\ No newline at end of file
+
+## API
+
+_The api documentation is in progress._
+
```

Comment:
We should add for which platform they are supported, in IOS `getCurrentWifiSSID`,` connectToSSID`, `connectToProtectedSSID` and` disconnectFromSSID` are only implemented at the moment

Reply:
I agree, I would add something like [android, ios] behind every method to state which platforms support what.